Cargo.toml

```diff
@@ -61,6 +61,7 @@ itertools = "0.13.0"
 leptix_primitives = { version = "0.2" }
 tree-ds = {version = "0.1.5", features = ["serde", "compact_serde"] }
 bimap = {version = "0.6.3", features = ["serde"] }
+walkdir = "2.5"
```

Comment:
I've added this to use directly, but it was already part of the dependencies in Cargo.lock.
Can't do without it, if adding a dependency is not desirable.